app-deployment-manager/test/deployment_delete/deployment_delete_test.go

```diff
@@ -65,3 +66,26 @@ func (s *TestSuite) TestDeleteDeploymentParentOnlyDeleteType() {
 
 	s.T().Logf("successfully deleted deployment with ID: %s using delete type", deploymentID)
 }
+
+func (s *TestSuite) TestDeleteDeploymentAllDeleteType() {
```

Comment:
That reason that I didn't add this because we need to see how we can check the actual behavior of this to distinguish it with other case PARENT_ONLY. My understanding is that this deletes all deployments in all targets based on this code https://github.com/open-edge-platform/app-orch-deployment/blob/main/app-deployment-manager/internal/northbound/northbound.go#L1148

Reply:
I see. I think we want to trigger case 3 here (https://github.com/open-edge-platform/app-orch-deployment/blob/main/app-deployment-manager/internal/northbound/northbound.go#L1202).